packages/dev/docs/pages/releases/2023-12-20.mdx

```diff
@@ -0,0 +1,277 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we’re promoting React Aria Components from RC to GA 🎉, optimizing our included translation files, shipping a storybook based on all our React Aria Components with a starter theme, and fixing plenty of bugs.
+
+date: 2023-12-20
+---
+
+# December 20, 2023 Release
+
+Welcome to the last release of the year! After a year of work, we are happy to announce the GA release of React Aria Components 🎉 This component library, built on top of our React Aria hooks, provides a simpler way to craft world-class accessible components with custom styles. See our new React Aria [landing page](https://react-spectrum.adobe.com/react-aria/index.html) for all of the details, including interactive examples. To help get you started even faster, we now also have a downloadable Storybook [starter kit](../react-aria/getting-started#starter-kit) including an example CSS theme.
+
+Also in this release we are promoting React Spectrum's DropZone to RC, adding the ability to optimize translation files, and fixing plenty of bugs.
+As always, huge thanks to everyone in our community for their feedback and contributions to make this release and this year possible!
+
+Happy coding and see you in the new year! ☃️
+
+
+### React Aria Components API updates
+
+In this release, we made a few last API updates to React Aria Components as they move along to GA.
+
+ - Temporarily removed nested column support in `Tables` due to instability.
+ - Our `Collections` have come a long ways since we started RAC, please see the new page, [React Aria Collections](../react-aria/collections) for more information.
```

Comment:
This link doesn't work.

```suggestion
 - Our `Collections` have come a long ways since we started RAC, please see the new page, [React Aria Collections](../react-aria/collections.html) for more information.
```

Reply:
no, but figured if anyone was coming from alpha.1 or something, might be good to call out